pkgs/development/compilers/sagittarius-scheme/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv
+, fetchurl
+, cmake
+, pkgconfig
+, libffi
+, boehmgc
+, openssl
+, zlib
+, odbc ? true
+, libiodbc
+}:
+
+let version = "0.9.6";
+    odbcDeps = if odbc then [ libiodbc ] else [];
+in stdenv.mkDerivation {
+  name = "sagittarius-scheme-${version}";
```

Comment:
Please replace with `pname = "sagittarius-scheme";` as `name = "${pname}-${version}";` is already implied.